elfi/methods/parameter_inference.py

```diff
@@ -177,27 +112,33 @@ def __init__(self, model, outputs, batch_size=1000, seed=None, pool=None,
         # State and objective should contain all information needed to continue the
         # inference after an iteration.
         self.state = dict(n_sim=0, n_batches=0)
-        self.objective = dict(n_batches=0)
+        self.objective = dict()
 
     @property
     def pool(self):
+        """Return the output pool of the inference."""
         return self.model.computation_context.pool
 
     @property
     def seed(self):
+        """Return the seed of the inference."""
         return self.model.computation_context.seed
```

Comment:
Not really because it's given in the beginning and should not be changed in between.